sdk/keyvault/TROUBLESHOOTING.md

```diff
@@ -93,10 +96,41 @@ The message and inner `code` may vary, but the rest of the text will indicate wh
 This error indicates that the authenticated application or user does not have permissions to perform that operation, though the cause may vary.
 
 1. Check that the application or user has the appropriate permissions:
-    * [Access policies](https://docs.microsoft.com/azure/key-vault/general/assign-access-policy) (Key Vault)
-    * [Role-Based Access Control (RBAC)](https://docs.microsoft.com/azure/key-vault/general/rbac-guide) (Key Vault and Managed HSM)
+   - [Access policies](https://docs.microsoft.com/azure/key-vault/general/assign-access-policy) (Key Vault)
+   - [Role-Based Access Control (RBAC)](https://docs.microsoft.com/azure/key-vault/general/rbac-guide) (Key Vault and Managed HSM)
 2. If the appropriate permissions are assigned to your application or user, make sure you are authenticating as that user.
-    * If using the [DefaultAzureCredential] a different credential might've been used than one you expected. [Enable logging](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/README.md#logging) and you will see which credential the [DefaultAzureCredential] used as shown below, and why previously-attempted credentials were rejected.
+   - If using the [DefaultAzureCredential] a different credential might've been used than one you expected. [Enable logging](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/README.md#logging) and you will see which credential the [DefaultAzureCredential] used as shown below, and why previously-attempted credentials were rejected.
```

Comment:
I don't understand this first sentence. It's not grammatically correct, but I'm also not sure I understand what you're trying to say. Please clarify.

Reply:
This area isn't new, and was just reformatted, so I didn't pay too much attention to it. We can do better, though, I agree.